LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -1,179 +1,301 @@
-                    throw e;
-                }
-                if(ldapClient.Bound)
-                {
-                    if(user == null)
-                    {
-                        // Determine if the user should be an administrator
-                        bool ldap_isAdmin = false;
-                        // Search the current user DN with the adminFilter
-                        LdapSearchResults ldapUsers = ldapClient.Search(ldapUser.DN, 0, adminFilter, ldapAttrs, false);
-                        var hasMore = ldapUsers.hasMore();
-                        // If we got non-zero, then the filter matched and the user is an admin
-                        if(ldapUsers.Count != 0)
-                        {
-                            ldap_isAdmin = true;
-                        }
-
-                        _logger.LogDebug("Creating new user {1} - is admin? {2}", ldap_username, ldap_isAdmin);
-                        if(_config.CreateUsersFromLdap)
-                        {
-                            user = _userManager.CreateUser(ldap_username);
-                            user.Policy.AuthenticationProviderId = GetType().Name;
-                            user.Policy.IsAdministrator = ldap_isAdmin;
-                            _userManager.UpdateUserPolicy(user.Id, user.Policy);
-                        }
-                        else
-                        {
-                            _logger.LogError($"User not configured for LDAP Uid: {ldap_username}");
-                            throw new Exception($"Automatic User Creation is disabled and there is no Jellyfin user for authorized Uid: {ldap_username}");
-                        }
-                    }
-                    return new ProviderAuthenticationResult
-                    {
-                        Username = ldap_username
-                    };
-                }
-                else
-                {
-                    _logger.LogError("Error logging in, invalid LDAP username or password");
-                    throw new Exception("Error completing LDAP login. Invalid username or password.");
-                }
-            }
-        }
-
-        public bool HasPassword(User user)
-        {
-            return true;
-        }
-
-        public string GetPasswordHash(User user)
-        {
-            return String.Empty;
-        }
-
-        public string GetEasyPasswordHash(User user)
-        {
-            return String.Empty;
-        }
-
-        public Task ChangePassword(User user, string newPassword)
-        {
-            throw new NotImplementedException("Changing LDAP passwords currently unsupported");
-        }
-
-        public void ChangeEasyPassword(User user, string newPassword, string newPasswordHash)
-        {
-            throw new NotImplementedException("EasyPin passwords for LDAP users are currently unsupported");
-        }
-    }
-}
+using System;
```

Comment:
Line ending changes?

Reply:
Not sure why it thinks the entire file is different, my file shows all line endings being `LF`